docs/unified-test-documentation/dasharo-security/200-tpm-support.md

```diff
@@ -404,3 +404,233 @@ Example output:
+    ```
+
+1. Note the result
+
+**Expected result**
+
+1. Output should look similar as below:
+
+    ```bash
+    name-alg:
+      value: sha256
+      raw: 0xb
+    attributes:
+      value: fixedtpm|fixedparent|sensitivedataorigin|userwithauth|restricted|decrypt
+      raw: 0x30072
+    type:
+      value: rsa
+      raw: 0x1
+    exponent: 65537
+    bits: 2048
+    scheme:
+      value: null
+      raw: 0x10
+    scheme-halg:
+      value: (null)
+      raw: 0x0
+    sym-alg:
+      value: aes
+      raw: 0x6
+    sym-mode:
+      value: cfb
+      raw: 0x43
+    sym-keybits: 128
+    rsa: b24f8e578d946a2157b1b442940fd5236bcc5041cfae37a56515eb28be8d7c06435d7356ce6c635c17cfc98031217fb462d94dc62821f0e0f2912012660305279a12a9359f23ba25cddb47f1e18d17b4af7bf8ef16d8daf65b9a5df8d669366ba2c9f7b187c64f59d6a79a3bb5b5f96e54529e69b3235311a87bf49c84b71cde1df5c6f82bf468653d9013c430044eb44ece988fecb59d4cfc37d4575d0dca1b68ad0893aa85eb8159c8c71dca0482e5915a28da456668a07a55dc5691e9c8863125e58d41de1cf7b6b97f35148a08c241a20571dfce16522b95f04a5be0fb5fff05c5075fb6d7921d87056169bb02adc31af257fbc512ebe0d95b18a3f94dad
+    ```
+
+## TPM010.001 NVDEFINE Function (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that NVDEFINE function works as expected.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+1. Install `tpm2-tools` using:
+
+    ```bash
+    sudo apt-get install tpm2-tools
+    ```
+
+    [Building from source](https://tpm2-tools.readthedocs.io/en/latest/INSTALL/)
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Execute command in the terminal:
+
+    ```bash
+    sudo tpm2_nvdefine 0x1500016
+    ```
+
+1. Note the result
+
+**Expected result**
+
+1. Output should look similar as below:
+
+    ```bash
+    nv-index: 0x1500016
+    ```
+
+## TPM011.001 NVUNDEFINE Function (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that NVUNDEFINE function works properly.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+1. Install `tpm2-tools` using:
+
+    ```bash
+    sudo apt-get install tpm2-tools
+    ```
+
+    [Building from source](https://tpm2-tools.readthedocs.io/en/latest/INSTALL/)
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Execute command in the terminal:
+
+    ```bash
+    sudo tpm2_nvundefine 0x1500016
+    ```
+
+1. Note the result
+
+**Expected result**
+
+1. Output should be empty. If region was undefined or error occured output
+    should look like this:
```

Comment:
And what does this output contains? How is anyone supposed to determine whether a feature is working or not?

Reply:
Updated